Source/WebKit/NetworkProcess/webrtc/NetworkRTCUDPSocketCocoa.mm

```diff
@@ -386,7 +386,7 @@ static inline void processUDPData(RetainPtr<nw_connection_t>&& nwConnection, Ref
         }).iterator->value.first.get();
     }
 
-    auto value = adoptNS(dispatch_data_create(data, size, nullptr, DISPATCH_DATA_DESTRUCTOR_DEFAULT));
+    RetainPtr value = adoptNS(dispatch_data_create(data.data(), data.size(), nullptr, DISPATCH_DATA_DESTRUCTOR_DEFAULT));
```

Comment:
Why isn't this still `auto` -- doesn't `adoptNS` mean we get a `RetainPtr`?

Reply:
We don't hide smart pointers behind `auto` anymore, to make it clear we're dealing with smart pointers. No behavior change though.